app/editor/src/components/form/toning/ToningGroup.tsx

```diff
@@ -57,7 +56,7 @@ export const ToningGroup: React.FC<IToningGroupProps> = ({ fieldName }) => {
           </Col>
         ))}
       </Row>
-      <Error error={!!fieldName && touched[fieldName] ? errors[fieldName] : ''} />
+      <Error error={!active && active !== 0 && touched[fieldName] ? 'Toning is required' : ''} />
```

Comment:
I don't understand the change here?  The original should display the error that was applied in the yup schema.

Reply:
The original schema is setup to track a number; however, tonePools is an array. So I changed the schema accordingly; however, and empty array counts as truthy. So I tried adding a minimum entries of 1 - but that was not triggering the validation appropriately. Going to reapproach it, but just putting this up so things can be saved in the meantime